pytorch/mnist/pytorch_mnist.py

```diff
@@ -6,16 +6,21 @@
 import engineml.torch as eml
 import numpy as np
 import pandas as pd
+import torch
 import torch.nn as nn
 import torch.nn.functional as F
 import torch.optim as optim
 import torch.utils.data.distributed
 from PIL import Image
 from tenacity import retry, stop_after_attempt, wait_fixed
-from tensorboardX import SummaryWriter
 from torch.autograd import Variable
 from torch.utils.data import Dataset
 
+if torch.__version__ >= '1.2.0':
+  from torch.utils.tensorboard import SummaryWriter
```

Comment:
https://tensorboardx.readthedocs.io/en/latest/tensorboard.html#tensorboardX.SummaryWriter

Maybe we just need to update `flush_secs`?

Reply:
I thought about going down the route of updating tensorboadX and trying to get it to work. But I saw issues like this in tensorboardX's github: https://github.com/lanpa/tensorboardX/issues/532

Also, doesn't it make more sense to just use pytorch's supported version?